src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -505,6 +515,64 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  'Boolean Subtract': {
+    description: 'Subtract one solid from another.',
+    icon: 'booleanSubtract',
+    needsReview: true,
```

Comment:
Do we also want to add a warningMessage and make it unavailable in the command bar to non-dev and non-nightly users?

Applies to all three

Reply:
Yup fixed

Reply:
It's just hidden now, so you basically can access it all outside of nightly atm.

Reply:
kcl is still there ofc, but Jess marked it as deprecated.